data/mods/Aftershock/afs_speech.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "type": "speech",
+    "speaker": "afs_mon_migoturret",
+    "sound": "\"Hello?\"",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": "afs_mon_migoturret",
+    "sound": "\"Can you help me?\"",
+    "volume": 40
+  },
+  {
+    "type": "speech",
+    "speaker": "afs_mon_migoturret",
+    "sound": "\"I have no mouth and yet I scream!\"",
+    "volume": 60
+  },
+  {
+    "type": "speech",
+    "speaker": "afs_mon_migoturret",
+    "sound": "\"Porque?\"",
```

```suggestion
    "sound": "\"Por que?\"",
```
porque is because, por que is why